tests/_fixtures/quota-management/roles.json

```diff
@@ -55,7 +55,6 @@
         "limit": {
           "cpus": 2.0,
           "disk": 10.0,
-          "gpus": 0,
```

Comment:
Why this is removed?

Reply:
there's an integration test that checks for `N/A`. that one broke when we implemented this fix, because it now would show `0`.

Reply:
i'll add another test that checks for `0`.